### DISCUSSION TITLE
If is ignored

### DISCUSSION THREAD

**Discussion by ostracizado** [Flexget/Flexget#3834] [Q&A] [answered]
```
web_server:
  bind: 0.0.0.0
  port: 5050
  web_ui: yes

tasks:
  specifictask:
    rss: https://rsslink
    accept_all: no
    if:
      - "'S02E' in title": accept
      - "'1080' in title": accept
    download: /data



schedules: no
```
Any ideas?

Even with "accept_all: no" defined, and the if inputs, it simply send all links (in my case) to deluge.

**Comment by gazpachoking** [comment]
Maybe you forgot to reload the daemon after editing your config, and used to have `accept_all: yes`? (`accept_all: no` doesn't do anything btw, the default state for entries is unaccepted)

**Reply by ostracizado** [reply]
Still no luck.

Config:
```
web_server:
  bind: 0.0.0.0
  port: 5050
  web_ui: yes

tasks:
 task_X:
    rss: https://wishlistturl
    if:
      - "'S01' in title": accept
      - "'1080p' in title": accept
    download: /data


schedules: no
```
I can confirm there are, at least, 2 files with this characteristics.

I forgot to say I was using flexplex docker's image - "linuxserver/flexget:latest".
I even restarted the docker container; gave another name to the task; delete config-lock and the bak files.

As client DelugeVPN - 2.1.1
Both on Portainer - 2.18.4.

Even the logs don't give much help:

Flexget WebUi:

```
TIME
LEVEL
PLUGIN
TASK
MESSAGE
2023-09-02 17:21:04
INFO
web_server
WebUI (v2) available at http://172.17.0.12:5050/
2023-09-02 17:21:04
INFO
web_server
API available at http://172.17.0.12:5050/api
2023-09-02 17:21:04
INFO
web_server
Web server started at http://172.17.0.12:5050
2023-09-02 17:21:04
INFO
web_server_daemon
Registering WebUI v2
2023-09-02 17:21:04
INFO
web_server_daemon
Initiating API
2023-09-02 17:21:04
INFO
web_server_daemon
Running web server at IP 0.0.0.0:5050
2023-09-02 17:20:31
INFO
web_server
Shutting down web server
```
...don't even register the manual task.

Portainer's log:
```
[custom-init] No custom files found, skipping...
2023-09-02 17:21:04 INFO     web_server_daemon                 Running web server at IP 0.0.0.0:5050
2023-09-02 17:21:04 INFO     web_server_daemon                 Initiating API
2023-09-02 17:21:04 INFO     web_server_daemon                 Registering WebUI v2
2023-09-02 17:21:04 INFO     web_server                    Web server started at http://172.17.0.12:5050
2023-09-02 17:21:04 INFO     web_server                    API available at http://172.17.0.12:5050/api
2023-09-02 17:21:04 INFO     web_server                    WebUI (v2) available at http://172.17.0.12:5050/
[ls.io-init] done.
```
The same.

The Flexget's History, of "Task Executions", gives this incongruent result of 24 links rejected but none accepted:
![Untitled-10](https://github.com/Flexget/Flexget/assets/78053830/36d80039-23a5-4c45-b4c9-380173fef80a)

**Comment by gazpachoking** [comment]
Ahh, I thought you meant everything was getting accepted still. I would run the task manually with `flexget --test execute --tasks specifictask --verbose --dump` in order to help debug what's going wrong and what is rejecting your entries. I'm guessing you ran it once at first with accept_all, and now the seen plugin is rejecting everything because it was already accepted once on a previous run. I'd also probably use the `regexp` plugin to accept things rather than the `if` plugin.

**Comment by BrutuZ** [comment]
I feel there are more ingredients in this soup that have been purged from the snippet you provided, including the culprit.

The `accepted_by` tag contains the name of the plugin that accepted the tag. You can either output it somewhere like a notification or if the CLI is an option, running the task manually with the `--dump` flag would be faster.
`flexget --test execute --task task_X --dump accepted trace`

**Comment by BrutuZ** [comment]
Might also be worth pointing out just for the sake of clarification that the `if` statements are not mutually exclusive.
An entry will be accepted if it has **either** `S01` **OR** `1080p` on its title, not necessarily both

**Comment by ostracizado** [comment]
```
web_server:
  bind: 0.0.0.0
  port: 5050
  web_ui: yes

tasks:
  task_a:
    rss: https://whislistrss
    regexp:
      accept:
        - S01
    download: /data


schedules: no
```

This seems to work; but limited to filtering/accepting only "S01". 
How should I go about filtering it to the two terms (ex: S01 & 1080p - only)?

**Reply by BrutuZ** [reply, marked as answer]
"Quick and dirty" RegEx: `S01.+1080p`
`.` is the wildcard character, will match anything, `+` is the "one or more" multiplier for the previous character. For reference: https://regexr.com

Another option is to use the [`quality` plugin](https://flexget.com/Plugins/quality)

**Reply by gazpachoking** [reply]
Are your requirements such that you don't want to just use the `series` plugin to accept episodes?
